docs/deployment-environment.mdx

```diff
@@ -27,6 +27,10 @@ After that, the previous deployment will start shutting down.
 This means that there is a window of time where there are two instances running in parallel.
 If the new deployment fails to become healthy, it will be retried 3 times while the the previous one will stays up.
 
+### Runtime architecture
+
+Unless you are on the Growth tier, your projects will primarily run on an Arm64 (aarch64) architecture, matching the [build architecture](/docs/builds#build-architecture).
```

Comment:
Same comment here:
```suggestion
Unless you are on the Growth tier, your projects will run on an Arm64 (aarch64) architecture, matching the [build architecture](/docs/builds#build-architecture).
```

Reply:
Hmm, I'm not sure. I would just drop it, for now. If there are exceptions, those should be documented, but I'm not sure if we have clarity on what the exceptions will be yet, e.g. allowing some tiers to opt out, so we can get back to that when we do.